src/it/scala/uk/gov/hmrc/govukfrontend/views/components/GovukBackLinkIntegrationSpec.scala

```diff
@@ -17,16 +17,39 @@
 package uk.gov.hmrc.govukfrontend.views
 package components
 
+import play.api.mvc.RequestHeader
+import play.api.test.FakeRequest
+import play.twirl.api.HtmlFormat
 import uk.gov.hmrc.support.ScalaCheckUtils.ClassifyParams
 import uk.gov.hmrc.govukfrontend.support.TemplateIntegrationSpec
+import uk.gov.hmrc.govukfrontend.views.GovukFrontendDependency.govukFrontendVersion
+import uk.gov.hmrc.govukfrontend.views.components.GovukNotificationBannerIntegrationSpec.app
 import uk.gov.hmrc.govukfrontend.views.html.components._
 import uk.gov.hmrc.govukfrontend.views.viewmodels.backlink.BackLink
 import uk.gov.hmrc.govukfrontend.views.viewmodels.backlink.Generators._
+import uk.gov.hmrc.helpers.MessagesSupport
+import uk.gov.hmrc.support.TemplateIntegrationBaseSpec
 
 import scala.collection.compat.immutable.LazyList
+import scala.util.Try
 
 object GovukBackLinkIntegrationSpec
-    extends TemplateIntegrationSpec[BackLink, GovukBackLink](govukComponentName = "govukBackLink", seed = None) {
+    extends TemplateIntegrationBaseSpec[BackLink](
+      componentName = "govukBackLink",
+      seed = None
+    )
+    with MessagesSupport {
+
+  protected val libraryName: String = "govuk"
+
+  protected val libraryVersion: String = govukFrontendVersion
+
+  private val component = app.injector.instanceOf[GovukBackLink]
+
+  override def render(notificationBanner: BackLink): Try[HtmlFormat.Appendable] = {
+    implicit val request: RequestHeader = FakeRequest("GET", "/foo")
```

Comment:
I think we might not need this implicit request since we have MessagesSupport trait mixed in - and if we remove it then we could change the param name too from notificationBanner